src/Analysis/GWASResults/Views/Results/ResultsPheWeb/ResultsPheWeb.jsx

```diff
@@ -21,6 +21,14 @@ const ResultsPheWeb = () => {
     () => getDataForWorkflowArtifact(name, uid, 'pheweb_json_index'),
     queryConfig,
   );
+  const [api, contextHolder] = notification.useNotification();
```

Comment:
I validated this works. Real good!
![image](https://github.com/uc-cdis/data-portal/assets/113449836/99ae4e37-cda2-4169-83fb-12c3038047a3)